### PR TITLE
#51 - implemented Debian.Asto.updatePackages

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #51
Implemented `Debian.Asto.updatePackages()`

### CODE REVIEW

File: src/test/java/com/artipie/debian/DebianTest.java [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+        final ByteArrayOutputStream out = new ByteArrayOutputStream();
+        try (GzipCompressorOutputStream gcos = new GzipCompressorOutputStream(out)) {
+            gcos.write(bytes);
+        }
+        return out.toByteArray();
+    }
+
+    private String pspp() {
+        return String.join(
+            "\n",
```
Comment (genryxy) [inline src/test/java/com/artipie/debian/DebianTest.java:172]: @olenagerasimova it looks too verbose. Probably to create some files for these three methods and after that read content of these files.

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one suggestion

**Comment by olenagerasimova** [comment]
@genryxy thanks, I've added todo, please, take a look

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**